resources/schema.gql

```diff
@@ -578,7 +603,9 @@ type Query {
 }
 
 enum QueryDialect {
-	DATA_FUSION
+	SQL_SPARK
+	SQL_FLINK
+	SQL_DATA_FUSION
```

Comment:
Note a small breaking change here. @dmitriy-borzenko will need to update the `{ data { query(queryDialect: ...) {} } }` parameter accordingly.

Motivation is to be start preparing for non-SQL dialects, e.g. `SPARQL`.